challenge-weather-app/index.html

```diff
@@ -1,49 +1,117 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
 
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-
-  <title>Meteoropolis</title>
-  <link rel="stylesheet" href="assets/globals.css">
-  <link rel="stylesheet" href="assets/styles.css">
-</head>
-
-<body>
-  <!-- Structure -->
-  <main class="content">
-    <header class="header">
-      <h1 class="title">
-        <i>Meteor</i>
-        <i>opolis</i>
-      </h1>
-    </header>
-
-    <figure class="photo" id="photo"></figure>
-
-    <div class="info">
-      <p class="info__item info__item--conditions" id="conditions"></p>
-      <p class="info__item info__item--credits">
-        <a href="#" id="credit-user"></a>
-        <span>on</span>
-        <a href="#" id="credit-platform">Unsplash</a>
-      </p>
-    </div>
-
-    <div class="thumbs" id="thumbs"></div>
-
-    <div class="controls">
-      <form class="search" id="search">
-        <label class="search__label" for="search-tf">City</label>
-        <input class="search__input" id="search-tf" name="city" placeholder="Enter city name" autocomplete="city" />
-        <button class="btn search__btn">Go</button>
-      </form>
-    </div>
-  </main>
-
-  <!-- JS goes here -->
-</body>
+    <title>Meteoropolis</title>
+    <link rel="stylesheet" href="assets/globals.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+  </head>
 
+  <body>
+    <!-- Structure -->
+    <main class="content">
+      <header class="header">
+        <h1 class="title">
+          <i>Meteor</i>
+          <i>opolis</i>
+        </h1>
+      </header>
+
+      <figure class="photo" id="photo"></figure>
+
+      <div class="info">
+        <p class="info__item info__item--conditions" id="conditions"></p>
+        <p class="info__item info__item--credits">
+          <a href="#" id="credit-user"></a>
+          <span>on</span>
+          <a href="#" id="credit-platform">Unsplash</a>
+        </p>
+      </div>
+
+      <div class="thumbs" id="thumbs"></div>
+
+      <div class="controls">
+        <form class="search" id="search">
+          <label class="search__label" for="search-tf">City</label>
+          <input class="search__input" id="search-tf" name="city" placeholder="Enter city name" autocomplete="city" />
+          <button class="btn search__btn">Go</button>
+        </form>
+      </div>
+    </main>
+
+    <!-- JS goes here -->
+
+    <script>
+      const apiKey = "2d2afc913f90548905b086803a05ef5d";
+      const accessKey = "m89STg41g63uXj4rt_9u5GXtN9LTGDH_ViVVX9Sy9Yc";
```

Comment:
Currently, we are storing keys in the codebase which is not secure as you can imagine. We will learn to store them somewhere else in the future. Please remember this. No need to any action for now- just info :)